src/pages/workspace/categories/WorkspaceCategoriesSettingsPage.tsx

```diff
@@ -44,7 +53,9 @@ function WorkspaceCategoriesSettingsPage({policy, route}: WorkspaceCategoriesSet
                 <View style={styles.flexGrow1}>
                     <ToggleSettingOptionRow
                         title={translate('workspace.categories.requiresCategory')}
-                        subtitle={isConnectedToAccounting ? `${translate('workspace.categories.needCategoryForExportToIntegration')} ${translate('workspace.accounting.qbo')}` : ''}
+                        titleStyle={styles.textStrong}
```

Comment:
[The text in the `ToggleSettingOptionRow` shouldn't be bold.](https://expensify.slack.com/archives/C01SKUP7QR0/p1715875595366199) 🙇